canonical-web-and-design/manager.assets.ubuntu.com
# Migrate to flask

Migrate the application to flask.

# QA
- set your `.env.local` as described in the `README`
- ./run
- make sure the site works
